Lib/typing.py

```diff
@@ -272,7 +272,7 @@ class _Final:
 
     __slots__ = ('__weakref__',)
 
-    def __init_subclass__(self, *args, **kwds):
+    def __init_subclass__(self, /, *args, **kwds):
```

Comment:
It was already mentioned above that it seems better to _only_ fix a small number of places where positional-only arguments are clearly what is needed.

In other cases it looks a bit cryptic. But IIUC it is not clear that we will go this way, maybe we will decide to use the per-argument syntax with underscores.